product/src/main/generated/transitive-dependencies-all.txt

```diff
@@ -2044,9 +2044,6 @@ org.apache.flink:flink-annotations
 org.apache.flink:flink-clients
 org.apache.flink:flink-connector-datagen
 org.apache.flink:flink-core
-org.apache.flink:flink-core-api
```

Comment:
not sure why it deletes those flink dependencies

Reply:
Earlier we are using flink community version with camel once moving to productized 1.19.1 version it got removed from the transitive-dependencies sections